src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: New runtime transition guide
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+Synthetic monitoring has a new runtime that affects your public monitors. By converting from the legacy runtime to the new runtime, you improve the backend environment that your public monitors execute from. New synthetic monitors default to using the new runtime. If you've created a synthetic monitor using the legacy runtime, there are extra [configuration steps](#runtime-convert).
```

Comment:
Should we say Synthetic monitoring has a new runtime available on public locations instead?

I think we can remove the sentence about converting from the legacy runtime to the new runtime too as we cover those benefits in the making the switch section below.